cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -69,9 +70,10 @@ func CreateAwsManager(configReader io.Reader) (*AwsManager, error) {
 
 	service := autoscaling.New(session.New())
 	manager := &AwsManager{
-		asgs:     make([]*asgInformation, 0),
-		service:  service,
-		asgCache: make(map[AwsRef]*Asg),
+		asgs:                     make([]*asgInformation, 0),
+		service:                  service,
+		asgCache:                 make(map[AwsRef]*Asg),
+		instancesNotInManagedAsg: make(map[AwsRef]struct{}),
```

Comment:
> I'm probably not the best person to make any calls here

So am I!
I guess that we are the only contributors concerned to the AWS part of CA? 😄 
As this is being an OSS project, I suppose that one of maintainers, or one of active contributors appointed as responsible by a maintainer, would be eligible make decision.